test/integration/test/no_travis/pagekite_access_test.js

```diff
@@ -1,96 +0,0 @@
-'use strict';
```

Comment:
No, I had to remove this test, because with the TLS implementation, the ping server will not provide the tunnel url, making this test obsolete.